mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -70,6 +70,9 @@
 # Maximum time to wait for a response message on the socket.
 _SOCKET_READ_TIMEOUT = callback_handler.MAX_TIMEOUT
 
+# Maximum logging length of Rpc response in DEBUG level.
```

Comment:
this does not seem accurate
should be something like "when verbose logging is off`?